frontend/templates/product/sections/CompatibilityNotesSection/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Heading, SimpleGrid } from '@chakra-ui/react';
+import { Wrapper } from '@ifixit/ui';
+import type { Product } from '@pages/api/nextjs/cache/product';
+
+export type CompatibilityNotesSectionProps = {
+   compatibilityNotes?: Product['compatibilityNotes'] | null;
```

Comment:
Is having the `?` and `| null` redundant? Is there a reason we are calling this component at all with a null compatibilityNotes?

Reply:
typescript was complaining